python-flask/README.md

```diff
@@ -0,0 +1,53 @@
+# Python Flask Example
+
+Data consumer example build using Python3 and [Flask](https://github.com/pallets/flask)
+
+## Setup
+
+```console
+$ pip3 install flask
+```
+
+## Running
+
+Running the server:
+
+```console
+$ python3 python-flask-example.py
+ * Serving Flask app "python-flask-example" (lazy loading)
+ * Environment: production
+   WARNING: This is a development server. Do not use it in a production deployment.
+   Use a production WSGI server instead.
+ * Debug mode: off
+ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
+----- Got message from D0123456789ABCDEF (YYYY-MM-DD HH:MM:SS.ffffff) -------
+Message path: /
+{'accelerometer': {'x': 0.123, 'y': 0.456, 'z': 0.789},
+ 'battery': 3.210,
+ 'gyroscope': {'x': 0.123, 'y': 0.456, 'z': 0.789},
+ 'magnetometer': {'x': 0.123, 'y': 0.456, 'z': 0.789},
+ 'neighbours': [{'id': 'D0123012301230123', 'rssi': -12},
+                {'id': 'DABCDABCDABCDABCD', 'rssi': -34},
+                {'id': 'D0011223344556677', 'rssi': -56}],
+ 'temperature': 24.321,
+ 'timestamp': 'YYYY-MM-DDTHH:MM:SS.ffffff',
+ 'usage': {'moving': 123, 'stationary': 456}}
+127.0.0.1 - - [DD/MONTH/YYYY HH:MM:SS] "POST / HTTP/1.1" 200 -
+```
+
+## Exposing a local server
+
+To test the setup on a local machine a tunneling service like [ngrok](https://ngrok.com/) can be used to expose the session. This will give you a URL to register in the Wittra Portal.
```

Comment:
Best practice to wrap at 80 characters :slightly_smiling_face: 

On another topic, should this really be part of how we use our API:s?

We can mention such services, of course, but to include information on how to use it just duplicates the services' own information (and may be outdated at some point in the future).

Reply:
Fixed.

Yes, that is a relevant discussion, it is nice to have it in one place now when this is the first and only example, but later this could be moved to some general hints/tools section..